backend/src/nodes/properties/outputs/numpy_outputs.py

```diff
@@ -96,10 +96,22 @@ def get_broadcast_data(self, value: np.ndarray):
         img = value
         h, w, c = get_h_w_c(img)
 
-        base64_img = preview_encode(img, 512)
+        previews = []
+
+        # Encode for multiple scales. Use the preceding scale to save time encoding the smaller sizes.
+        last_encoded = img
+        for size in [2048, 1024, 512, 256]:
+            if h >= size or w >= size:
```

Comment:
This condition isn't correct. Suppose we have an image 700x700px. This image will only get 512px and 256px previews, but no 700px preview.